indico_toolkit/pipelines/file_processing.py

```diff
@@ -44,6 +47,39 @@ def get_file_paths_from_dir(
             f"Found {len(self.file_paths)} valid files and {len(self.invalid_suffix_paths)} paths with invalid suffixes."
         )
 
+    def fix_file_suffixes(
```

Comment:
is this renaming the files in place on the system??

Reply:
think that should be clearer or optional

Reply:
I would also probably be more opinionated with how the function works, i.e. I don' think we need to give the option to pass a mime_suffix_mapping (I would have no idea what to do with this)